docs/Using-Fleet/REST-API.md

```diff
@@ -17,6 +17,7 @@
 - [Teams](#teams)
 - [Translator](#translator)
 - [Users](#users)
+- [Devices](#devices)
```

Comment:
I created https://github.com/fleetdm/fleet/issues/5968 to add documentation for the other device-related endpoints.